docs/content/docs/_index.md

```diff
@@ -35,9 +35,9 @@ Applications built with Fluid Framework require zero custom code on the server t
 scenarios such as real-time typing across text editors. Client developers can focus on customer experiences while
 letting Fluid do the work of keeping data in sync.
 
-Fluid Framework works with your application framework of choice. Whether you prefer straight JavaScript or
-a framework like React, Angular, or Vue, Fluid Framework makes building collaborative experiences simple and
-flexible.
+Fluid Framework works with your application framework of choice.
+Whether you prefer straight JavaScript or a framework like [React](https://reactjs.org), [Angular](https://angular.io),
```

Comment:
I think "plain" or "vanilla" JS is the more standard name here, but this might just be personal preference.

```suggestion
Whether you prefer plain JavaScript or a framework like [React](https://reactjs.org), [Angular](https://angular.io),
```